cmd/agent/app/integrations.go

```diff
@@ -599,7 +599,11 @@ func minAllowedVersion(integration string) (*semver.Version, error) {
 	}
 	version, err := getVersionFromReqLine(integration, string(lines))
 	if err != nil {
-		return nil, err
+		if err.Error() == fmt.Sprintf("unknown integration '%s'", integration) {
```

Comment:
returning an extra boolean in `getVersionFromReqLine` to indicate that the integration is not installed would be better that looking at the error message. Or returning a specific error type.